### PR TITLE
dialer/resolver - add ResolveAll, update fs_util

### PR DESCRIPTION
**Pull request by dgdonut**
bump fs_util version and add ResolveAll to dialer.Resolver. Pass all apiserver addrs to bz_tree operations via ResolveAll.

(should be considered a WIP for now)

### CODE REVIEW

File: common/dialer/resolver.go [hunk 1]
```diff
@@ -41,6 +52,16 @@ func (r *EnvResolver) Resolve() (string, error) {
 	return os.Getenv(r.key), nil
 }
 
+// ResolveAll retuns the env key in a slice
+func (r *EnvResolver) ResolveAll() ([]string, error) {
+	all := []string{}
+	s, err := r.Resolve()
+	if s != "" {
+		all = append(all, s)
+	}
```
Comment (ryancouto) [inline common/dialer/resolver.go:63]: should we return an err if s == ""?

File: common/dialer/resolver.go [hunk 2]
```diff
@@ -26,6 +28,15 @@ func (r *ConstantResolver) Resolve() (string, error) {
 	return r.s, nil
 }
 
+// ResolveAll returns the constant in a slice
+func (r *ConstantResolver) ResolveAll() ([]string, error) {
+	all := []string{}
+	if r.s != "" {
+		all = append(all, r.s)
+	}
```
Comment (ryancouto) [inline common/dialer/resolver.go:38]: ditto re: s == ""

File: common/dialer/resolver.go (hunk 1 above)
Comment (dgdonut) [reply common/dialer/resolver.go:63]: Not 100% clear from the interface, but I think it's ok to return an empty result, where empty is "" for Resolve and []string{} for ResolveAll, if the resolution logic was successful and there were just no results. For these simple resolvers that essentially means no errors, but gives room to error in more complicated resolvers e.g. service discovery lookups where an http call can fail.

Comment (ryancouto) [reply common/dialer/resolver.go:63]: Fair - makes sense.

**Review by ryancouto — APPROVED** [review]

**Pull request merged**